README.md

```diff
@@ -39,9 +39,13 @@ import threescale.v3.api.impl.*;
 
 //  ... somewhere inside your code
 
-// Create the API object
+// Create the API object with default settings. This will communicate with the 3scale platform SaaS default server.
 ServiceApi serviceApi = ServiceApiDriver.createApi();
 
+// Create the API object for a given host and port. Use this method when connecting to an on-premise instance of the 3scale platform. 
+ServiceApi serviceApi = ServiceApiDriver.createApi("auth.backend.com", 80, true);
```

Comment:
It is recommended to use `example.com` for examples of domains. So `backend.example.com` would be just fine.